libs/logging/src/logger.ts

```diff
@@ -30,25 +20,16 @@ interface LogData extends Dictionary<string | boolean | number> {
   disposition?: LogDisposition;
 }
 
-export function extractAdditionalKeysFromObj(
-  innerObj: Dictionary<string>,
-  outerObj: Dictionary<string>
-): Dictionary<string> {
-  const baseDict: Dictionary<string> = {};
-  return Object.keys(outerObj)
-    .filter((key) => !(key in innerObj))
-    .reduce((res, nextKey) => {
-      res[nextKey] = outerObj[nextKey];
-      return res;
-    }, baseDict);
-}
-
 export class Logger {
   constructor(
     private readonly source: LogSource,
     private readonly kiosk?: KioskBrowser.Kiosk
   ) {}
 
+  getSource(): LogSource {
```

Comment:
Because I extracted `buildCDFLog` as a standalone function and it needs access to `Logger::source`, I added this getter method.